Yoast/YoastSEO.js
# Add Dutch passive voice

## Summary

This PR can be summarized in the following changelog entry:

* Adds the passive voice assessment for Dutch.

## Relevant technical choices

## Test instructions

This PR can be tested by following these steps:

* Build a beta script using `trunk` for Yoast SEO and this branch for YoastSEO.js.
* Copy and paste a Dutch text from e.g. Wikipedia. See if passive voice is detected (if there are no passive sentences, this might also be due to the text; try a different text in that case).
* Make sure the following kinds sentences are marked as passive:
  * Passive sentences with the auxiliary `worden`, e.g. `Een appel wordt gegeten`.
* Make sure the following kinds of sentences are _not_ marked as passive:
  * Non-passive sentences, e.g. `Ik eet een appel.`

Fixes #1017
